gridgencore.py

```diff
@@ -418,9 +424,24 @@ class SingleGridCall:
     def __init__(self, values: list):
         self.values = values
         self.skip = False
+        skipDict = {'title': [], 'params': []}
         for val in values:
             if val.skip:
                 self.skip = True
+            if hasattr(val, 'skipList') and isinstance(val.skipList, dict):
```

Comment:
You again committed code without testing it

Reply:
You MUST test BEFORE committing.

Reply:
What? You committed a change that changed some copies of the field to `skip_list` but left several (as in the line here) as `skipList`. ie this field no longer functions as intended after your last commit.

The angry combative attitude continues to be unproductive.